es-design-system/layouts/nav.vue

```diff
@@ -1,6 +1,8 @@
 <template>
     <div>
-        <es-nav-bar />
+        <es-nav-bar
+            :account-content="accountContent"
+            :global-content="globalContent" />
```

Comment:
Might be out of scope but feels like `fetch('/api/account-first-name/',` could also be a prop here and give us an easier path to changing the logic.

```javascript

props: {
  isUserLoggedIn: {
    type: Function,
    required: false,
    default: () => false,
  }
}
```

Reply:
I think that is out of scope for this ticket. I also don't know a way of implementing that without breaking the ripped `es-site` version. Some possibilities:
1. Always make the hardcoded fetch API call and ignore the result if some prop is passed in (but then the API call would be made unnecessarily on HPM/CSM).
2. Have some second boolean prop defaulted to true that represents whether to make the hardcoded fetch API call or not, and in the non-Vue case, somehow do a whole bunch of existence checking to get around the fact that the `this.[propName]` reference will break in vanilla JS? Not sure.

Either way I think this should wait until the ticket to make the nav HPM/CSM compatible.

Reply:
With flash now live this is a lot easier; https://energysage.atlassian.net/browse/CED-511